lib/stripe/utils/index.js

```diff
@@ -1,18 +1,14 @@
 'use strict';
 var Promise = require('promise');
-var VATRates = require('./vat-rates.json').data;
+var VATRates = require('./vat-rates.json').rates;
```

Comment:
@allouis in case you were wondering...this one line was why it wasn't working :cry: